ibis/expr/api.py

```diff
@@ -2888,6 +2888,48 @@ def _string_join(self, strings):
     return ops.StringJoin(self, strings).to_expr()
 
 
+def _startswith(self, start):
+    """
+    Determine if `self` string starts with `start` string.
+
+    Parameters
+    ----------
+    start: string
+
+    Examples
+    --------
+    >>> import ibis
+    >>> text = ibis.literal('Ibis project)
+    >>> result = text.startswith('Ibis')
+
```

Comment:
can you add some examples here (and in endswith)

Reply:
You mean I should mention the result? Because above is showing an example on how to use or?

Reply:
Yes, you should write the output. Not sure if we're running these doctests at the moment (we do run some), but they'll fail if/when we do.

Reply:
Ok I updated this! Hope that's what you expected?

Reply:
@jreback are you happy with my changes to the example? Or did you expect a different format. Didnt really see many examples for this